tools/lint/src/main/java/com/grab/lint/Sanitizer.kt

```diff
@@ -0,0 +1,65 @@
+package com.grab.lint
+
+import org.apache.commons.io.output.StringBuilderWriter
+import java.io.File
+
+class Sanitizer(
+    private val env: Env = Env.BazelEnv
+) {
+
+    fun sanitize(file: File): StringBuilderWriter {
```

Comment:
- `sanitize(file).toString()` seems like a repeating pattern, instead you could sanitize the file in place without returning a string. 
- `StringBuilderWriter.toString()` brings the entire baseline content string which is huge to memory which is huge, instead you could directly write using a buffered text writer as done [here](https://github.com/grab/grab-bazel-common/blob/4db9d71787fcfd546412382c4cdcc2ad25d7992e/tools/lint/src/main/java/com/grab/lint/LintBaselineHandler.kt#L25). `useLines` already provides a lazy sequence.

Reply:
Yes, this way uses more RAM, but the suggested way means I should copy the sanitized version in a different file and then copy line by line, which will effect performance.